src/ListTabulator/index.ts

```diff
@@ -44,19 +37,19 @@ export default class ListTabulator {
   private data: ListData;
 
   /**
-   * Current level of nesting for dynamyc updates
+   * Editor block api
    */
-  private currentLevel: number;
+  private block: BlockAPI;
 
   /**
-   * Style of the nested list
+   * Current level of nesting for dynamyc updates
    */
-  style: ListDataStyle;
+  private currentLevel: number;
```

Comment:
It's not clear for me, why "currentLevel" is a property of Tabulator? It can be a property of a list item, or a sub-list, but Tabulator has to level